src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -57,7 +57,10 @@ const ImageWithPlaceholder = ({
           />
         </Helmet>
       )}
-      <ImagePlaceholder css={isLoaded && `background: none`} ratio={ratio}>
+      <ImagePlaceholder
+        style={{ background: isLoaded && 'none' }}
```

Comment:
If `isLoading` equals `false` this expression will return `false` – that could in turn mean we set `background: false`.

I think React is intelligent enough to omit Boolean values, but I would not leave this to chance. If you pass it a Number it actually stringifies and appends `px` to it! See https://reactjs.org/docs/dom-elements.html#style.

We could instead return `{ background: 'none' }` if `isLoading` is truthy e.g. `style={isLoading ? {} : null}`

Incidentally, https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx is also a good one on this topic.

Reply:
You're right, this is what I was trying to do. Just couldn't figure out the syntax, danke 👍